obsidian.d.ts

```diff
@@ -862,131 +2971,334 @@ export interface Command {
      *   }
      * });
      * ```
+     *
      * @public
      */
     editorCheckCallback?: (checking: boolean, editor: Editor, ctx: MarkdownView | MarkdownFileInfo) => boolean | void;
     /**
      * Sets the default hotkey. It is recommended for plugins to avoid setting default hotkeys if possible,
      * to avoid conflicting hotkeys with one that's set by the user, even though customized hotkeys have higher priority.
+     *
+     * @example
+     * ```ts
+     * this.addCommand({
+     *   id: 'example-command',
+     *   name: 'Example command',
+     *   hotkeys: [{
+     *     modifiers: ['Mod', 'Shift'],
```

Comment:
We explicitly ask plugins to not set a default hotkey for commands (see the doc string right above), so the example should also not set one.

Reply:
The idea was to provide some usage example, otherwise why would you have `hotkeys` at all. But I can add a comment repeating that it's a not recommended way and added only for example purposes.